src/code-actions/code-action-wrap.ts

```diff
@@ -2,15 +2,31 @@
 
 import { CodeAction, CodeActionKind, CodeActionProvider, window } from "vscode";
 import { getSelectedText } from "../utils";
+import { CustomWrapConfig } from "../config/config";
 
 export class CodeActionWrap implements CodeActionProvider {
+    // create constructor that takes Array<CustomWrapConfig> 
+    // as a parameter and assign it to a private field
+    private customWraps: Array<CustomWrapConfig>;
+
+    constructor(customWraps: Array<CustomWrapConfig>) {
+        this.customWraps = customWraps;
+    }
+
     public provideCodeActions(): CodeAction[] {
         const editor = window.activeTextEditor;
         if (!editor) return [];
 
         const selectedText = editor.document.getText(getSelectedText(editor));
         if (selectedText === "") return [];
 
+        const customCommands = this.customWraps.map((wrap) => {
+            return {
+                command: wrap.id,
```

Comment:
Maybe just transform `wrap.name` to kebab case and make it like
`flutter-plus.user.${wrap-kebab-name}`
?